pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -401,15 +394,77 @@ ARG PT_TORCHDATA_URL
 ARG PYTORCH_VERSION_SM
 ARG NCCL_ASYNC_ERROR_HANDLING
 
-# Install sm PyTorch (force remove will increase docker image size drastically)
+# Install ec2 PyTorch
```

Comment:
Please change this to "sm" PyTorch after the tests all run and the PR has temporary changes removed.